files/galaxy/dynamic_job_rules/pawsey/total_perspective_vortex/default_tool.yml.j2

```diff
@@ -0,0 +1,52 @@
+global:
+  default_inherits: default
+
+tools:
+  default:
+    cores: 1
+    mem: cores * 3.8
+    env: {}
+    params:
+      nativeSpecification: "--nodes=1 --ntasks={cores} --ntasks-per-node={cores} --mem={round(mem*1024)}"
+      submit_native_specification: "--nodes=1 --ntasks={cores} --ntasks-per-node={cores} --mem={round(mem*1024)}"
+    scheduling:
+      reject:
+        - offline
+    rules: []
+    rank: |
+      import requests
+      grafana_destinations = {
+        'Galaxy-Main': 'slurm',
+        'pulsar-high-mem-1': 'pulsar-high-mem1',
+        'pulsar-high-mem2': 'pulsar-high-mem2',
+        'pulsar-mel2': 'pulsar-mel2',
+        'pulsar-mel3': 'pulsar-mel3',
+        'pulsar-nci-training': 'pulsar-nci-training',
+        'pulsar-paw': 'pulsar-paw',
+        'pulsar-qld-himem-0': 'pulsar-qld-high-mem0',
+        'pulsar-qld-himem-1': 'pulsar-qld-high-mem1',
+        'pulsar-qld-himem-2': 'pulsar-qld-high-mem2',
+      }
+      params = {
+        'pretty': 'true',
+        'db': 'queues',
+        'q': 'SELECT last("percent_allocated") from "sinfo" group by "host"'
+      }
+      if len(candidate_destinations) > 1:
+        try:
+          response = requests.get('https://stats.usegalaxy.org.au:8086/query', auth=('grafana', '{{ vault_influx_grafana_password }}'), params=params)
```

Comment:
Maybe we could read the `vault_influx_grafana_password` off os.environ?

Reply:
the way that the galaxy user is set up, it has its own .bashrc file so it could have export INFLUX_GRAFANA_PASSWORD=xyz